web/template/certificateprovider/confirm_dont_want_to_be_certificate_provider.gohtml

```diff
@@ -0,0 +1,29 @@
+{{ template "page" . }}
+
+{{ define "pageTitle" }}{{ tr .App "confirmYouDoNotWantToBeACertificateProvider" }}{{ end }}
+
+{{ define "main" }}
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+            <h1 class="govuk-heading-xl">{{ tr .App "confirmYouDoNotWantToBeACertificateProvider" }}</h1>
+
+            {{ $donorFullNamePossessive := tr .App (possessive .App .Lpa.Donor.FullName) }}
+            {{ $lpaType := tr .App .Lpa.Type.String }}
+
+            {{ trFormatHtml .App "confirmYouDoNotWantToBeACertificateProviderContent" "DonorFullNamePossessive" $donorFullNamePossessive "LpaType" $lpaType "LpaUID" .Lpa.LpaUID }}
+
+            {{ template "details" (details . "ifYouHaveConcernsAboutThisLPA" "ifYouHaveConcernsAboutThisLPAContent" false) }}
+
+            <p class="govuk-body">{{ trFormat .App "whenYouSelectConfirmWeWillContactDonor" "DonorFirstNames" .Lpa.Donor.FirstNames }}</p>
+
+            <form novalidate method="post">
+                <div class="govuk-form-group">
+                    <fieldset class="govuk-fieldset">
```

Comment:
you don't need a form-group or fieldset for just a button

Reply:
Soz, copypasta